bin/install/upload_to_s3.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+COSMOS_VERSION="0.4.0"
+
+cosmos_cli()
+{
+        result=$(echo -e "$COSMOS_VERSION\n$TAG_VERSION" | sed '/^$/d' | sort -nr | head -1)
+        [[ "$result" = "$TAG_VERSION" ]]
+}
+
+if cosmos_cli ; then
+        aws s3 --region=us-east-1 cp \
+        %aws.bash_script_to_upload% \
+        %aws.bash_destination_url%
+
+        aws s3 --region=us-east-1 cp \
+        %aws.powershell_script_to_upload% \
+        %aws.powershell_destination_url%
+else
+        aws s3 --region=us-east-1 cp \
+        %aws.bash_script_to_upload% \
```

Comment:
I'm using `aws.bash_destination_legacy_url` for the s3 location where non-cosmos cli releases will be uploaded (which will be in a different location than cosmos cli releases)